packages/authentication/modules/access/allauth/useAllAuthLogin/constants.ts

```diff
@@ -2,14 +2,14 @@ import { ZOD_MESSAGE } from '@baseapp-frontend/utils'
 
 import { z } from 'zod'
 
-import type { LoginRequest } from '../../../types/auth'
+import * as AllAuthTypes from '../../../../types/allAuth'
```

Comment:
let's add the type directive to avoid including unused runtime code and reduce bundle size

```suggestion
import type * as AllAuthTypes from '../../../../types/allAuth'
```